### ISSUE TITLE
Media Export Feature Request: Provide users with VideoPress export option

### ISSUE THREAD

**Issue by JoshuaGoode** [Automattic/wp-calypso#67002]
### What

Offer Simple, WPCOM Atomic, and self-hosted Jetpack users the ability to export VideoPress videos via Calypso.

pxWta-1xi-p2#comment-5893

### Why

The VideoPress team recently released a new export tool. pxWta-1xi-p2 -- this addresses a gap we've noticed numerous times in support.

Currently, it requires contacting support and an HE using an internal tool.

Let's expose this to users.

### How

Implement VideoPress export option under the current Export tools.

Expose this to Simple, WPCOM Atomic, and self-hosted Jetpack users.

May also need to implement the media export tool on Atomic, remove redirects preventing access to the page. 

- [ ] https://github.com/Automattic/wp-calypso/issues/58466

**Comment by zachspears** [comment]
I strongly agree with @JoshuaGoode. This would also save time for HE's and provide a frictionless experience for users. Set low priority due to low impact.
